src/framework/mlt_repository.c

```diff
@@ -336,12 +336,13 @@ void *mlt_repository_create(mlt_repository self,
 
 void mlt_repository_close(mlt_repository self)
 {
-    mlt_properties_close(self->consumers);
-    mlt_properties_close(self->filters);
-    mlt_properties_close(self->producers);
-    mlt_properties_close(self->transitions);
-    mlt_properties_close(&self->parent);
-    free(self);
+	mlt_properties_close( self->consumers );
+	mlt_properties_close( self->filters );
+	mlt_properties_close( self->links );
+	mlt_properties_close( self->producers );
+	mlt_properties_close( self->transitions );
+	mlt_properties_close( &self->parent );
+	free( self );
```

Comment:
Indentation of this is still wrong; it uses 4 spaces now instead of tabs. Our `.clang-format` file covers that.